app/ix/core/models/Structure.java

```diff
@@ -184,6 +184,27 @@ public static Optical forValue(String value) {
     @Indexable(name = "Molecular Formula", facet = true)
     public String formula;
 
+    @JsonProperty("_html_formula")
+    public String htmlFormula() {
+        if (formula == null) {
+            return "";
```

Comment:
should this return `null` ?

Reply:
i prefer to see the empty string instead of null in json.